pom.xml

```diff
@@ -45,6 +46,11 @@
       <artifactId>jackson-databind</artifactId>
       <version>${fasterxml.jackson.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.dataformat</groupId>
+      <artifactId>jackson-dataformat-yaml</artifactId>
+      <version>${fasterxml.jackson.dataformat.yaml}</version>
```

Comment:
This library depends on jackson-databind 2.9.9. Wouldn't it be better to use the same variable ${fasterxml.jackson.version} but set to 2.9.9?

Reply:
good point! updated. thanks